.idea/jsonSchemas.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="JsonSchemaMappingsProjectConfiguration">
+    <state>
+      <map>
+        <entry key="config.yml">
+          <value>
+            <SchemaInfo>
+              <option name="name" value="config.yml" />
```

Comment:
Können wir das auch für diese datei nutzen?
https://github.com/redaxo/redaxo/blob/master/redaxo/src/core/default.config.yml

Reply:
Hatte ich erst. Dann müssten wir für instname und error_email auch `null` akzeptieren, was in der eigentlichen `config.yml` nicht erlaubt ist (und dann aber nicht mehr als invalid markiert werden würde).
War mir unsicher, was da nun besser ist.

Reply:
Fände ich nicht so schlimm..  ich denke da schreibt keiner bewusst null rein :)